src/mocks/CatalogueCategory.json

```diff
@@ -132,7 +142,8 @@
       {
         "name": "Wavefront Measurement Range",
         "type": "string",
-        "mandatory": true
+        "mandatory": true,
+        "allowed_values": null
       },
       {
         "name": "Spatial Resolution",
```

Comment:
This is missing "allowed_values", there are probably a few others. I searched for `type"` and compared with `allowed_values"` and it appears there are 6 missing in total.